fc/jira/task.py

```diff
@@ -4,77 +4,9 @@
 from ..auth.auth import Auth
 from typing import Optional
 from ..exceptions.task_exception import TaskException
+from .issue import Issue
 
-
-class Task:
```

Comment:
So I removed a lot of stuff that seems to be basic to issues and moved it to the `Issue` class.